debian/precise/foreman/foreman-debug.dirs

```diff
@@ -0,0 +1 @@
+usr/share/foreman/script/foreman-debug.d/
```

Comment:
This should be part of the main package if the proxy is going to drop files into it.